ios/Handlers/RNTapHandler.m

```diff
@@ -12,11 +12,22 @@
 
 #import <React/RCTConvert.h>
 
-@interface RNBetterTapGestureRecognizer : UIGestureRecognizer
+@interface RNBetterTapGestureRecognizer : UIPanGestureRecognizer
+// We decided to extend UIPanGestureRecognizer
```

Comment:
This comment would look better in an impersonal form:

> `RNBetterTapGestureRecognizer` extends `UIPanGestureRecognizer` instead of `UITapGestureRecognizer` so that translationInView can be used to make the recognizer respect max deltas constraint. Also parameters like `maxDelay` and `maxDuration` are not configurable in `UITapGestureRecognizer` and therefore require custom implementation.

Also would be better to put that comment above the interface definition